### PR TITLE
Add Blacklisted tab and count hours accurately

### PR DESCRIPTION
**Pull request by jasozh**
## Summary

- Add Blacklisted tab
- Add accurate hours calculations in
- Edit manage attendees message
- Fix select background
- Fix hours in ManageUsers table

### CODE REVIEW

File: backend/src/users/controllers.ts [hunk 1]
```diff
@@ -240,11 +241,23 @@ const getUsers = async (
...
   });
+
+  // Get hours for each user
+  const newQueryResult: any = queryResult;
+  for (let i = 0; i < queryResult.length; i++) {
+    const user = queryResult[i];
+    const hours = await getHours(user.id);
+    newQueryResult[i].totalHours = hours;
+  }
+
```
Comment (akinfelami) [inline backend/src/users/controllers.ts:252]: I thought we only want to grab the hours if the query specifies?

File: frontend/src/components/organisms/ManageUserProfileNew.tsx [hunk 2]
```diff
@@ -143,7 +153,7 @@ const ManageUserProfileNew = () => {
     email,
     joinDate,
     userId,
-    hours,
+    hours: fakeHours,
```
Comment (akinfelami) [inline frontend/src/components/organisms/ManageUserProfileNew.tsx:156]: What's this about?

**Review by akinfelami — CHANGES_REQUESTED** [review]
Great!

Two quick comments and we should be able to merge!

File: frontend/src/components/organisms/ManageUserProfileNew.tsx (hunk 2 above)
Comment (jasozh) [reply frontend/src/components/organisms/ManageUserProfileNew.tsx:156]: I can't delete the `hours` column from the `UserProfile` column, so we still grab the hours from the profile but we can't use it. I use `hours` for the actual hours and `fakeHours` for the hours grabbed from profile.

File: backend/src/users/controllers.ts (hunk 1 above)
Comment (jasozh) [reply backend/src/users/controllers.ts]: fixed

**Review by akinfelami — APPROVED** [review]
lgtm

**Pull request merged**